src/facades.php

```diff
@@ -111,3 +112,11 @@ function graphql(
         $fieldResolver
     );
 }
+
+/**
+ * @return ExecutionInterface
+ */
+function executor(): ExecutionInterface
```

Comment:
I think this method is unnecessary.

Reply:
My test simply doesn't work with function `graphql(SchemaInterface $schema, string $source,...`. It looks unnecessary, but it's not. Let's I think a bit.